content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -303,6 +304,21 @@ policy option is not compatible with `TopologyManager` `single-numa-node`
 policy and does not apply to hardware where the number of sockets is greater
 than number of NUMA nodes.
 
+
+If the `distribute-cpus-across-cores` policy option is specified, the static policy
+will attempt to allocate cpus (hyperthreads) across different physical cores.
+By default, the `CPUManager` tends to pack cpus onto as few physical cores as possible,
+which can lead to contention among cpus on the same physical core and result
+in performance bottlenecks. By enabling the `distribute-cpus-across-cores` policy,
+the static policy ensures that cpus are distributed across as many physical cores
+as possible, reducing the contention on the same physical core and thereby
+improving overall performance. However, it is important to note that this strategy
+might be less effective when the system is heavily loaded. Under such conditions,
+the benefit of reducing contention diminishes. Conversely, default behavior
+can help in maintaining cache locality and reducing inter-core communication
```

Comment:
This is confusing. The default policy uses as fewer cores as possible. That doesn't lead to "maintaining cache locality".

Reply:
Let's say one application requests 4 cpus. the default policy will help allocate 2 full physical cores and new policy allocate 4 cpus on 4 different physical cores. For the 1st option, two cpus shared the same physical cores have high cache locality.

Reply:
I think @Jeffwan is referring to L2 cache locality. Tipically L2 cache blocks are shared between virtual threads (example for AMD zen: https://www.servethehome.com/amd-epyc-genoa-gaps-intel-xeon-in-stunning-fashion/amd-epyc-9004-genoa-zen-4-cache-hierarchy/), while L3 cachec blocks are shared between physical cores.

I'd rather avoid the mention caches, just mentioning inter-core communication, or clarify we are hinting to core-level cache blocks

Reply:
Sounds good. Let me remove the cache part